### PR TITLE
product page

### PR DESCRIPTION
**Pull request by utkarshducoder**
Please review my products page

live link: https://sportssite.netlify.app/html/product.html

this page contains the following features:

- nav: contains the brand name and link to the home page.
- sidebar: conatins links to all of those filters available for the products.
- cards: contains links to all the categores of the sports.

f1

### CODE REVIEW

File: categories/product.css [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+  
+  .centered {
+    position: absolute;
+    top: 85%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    font-size: large;
+    font-weight: bold;
+    font-size: 80px;
+    color: white;
```
Comment (khushijohri2001) [inline categories/product.css]: use var here

**Review by khushijohri2001 — APPROVED** [review]

File: categories/product.css [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+      position: relative;
+      line-height: 60px;
+      color: #555;
+  }
+
+* {
+    --primary-color: hotpink;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+      Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+      margin: 0;
```
Comment (i-am-a-codegirl) [inline categories/product.css]: do proper allignment

File: categories/product.css [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+    margin: 0.3rem;
+  }
+  
+  .card-content {
+    margin: 0.5rem;
+  }
+  
+  .card {
+    background-color: rgb(104, 104, 204);
+    /* box-shadow: var(--box-shadow-card); */
```
Comment (i-am-a-codegirl) [inline categories/product.css]: remove unwanted comments

File: categories/product.css [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+  line-height: 80px;
+  margin-right: 40px;
+  cursor: pointer;
+  display: none;
+}
+#check{
+  display: none;
+}
+  @media (max-width: 480px){
+  label.logo{
```
Comment (i-am-a-codegirl) [inline categories/product.css]: shift all the media queries in the end

**Review by i-am-a-codegirl — APPROVED** [review]

**Pull request merged**